src/renderer/lib/telemetry.js

```diff
@@ -2,27 +2,31 @@
 // Reports back so that we can improve WebTorrent Desktop
 module.exports = {
   init,
+  send,
   logUncaughtError,
   logPlayAttempt
 }
 
-const crypto = require('crypto')
 const electron = require('electron')
-const https = require('https')
-const os = require('os')
-const url = require('url')
 
 const config = require('../../config')
 
 let telemetry
 
 function init (state) {
   telemetry = state.saved.telemetry
```

Comment:
Posting to the server used to happen in `init()` which did a lot of work gathering stats. Now we do that at the last minute in `send()` which is called after a delay.